gdb-xtensa-esp32-elf/meta.yaml

```diff
@@ -1,32 +1,31 @@
 {% set name = "xtensa-esp32-elf-gdb" %}
-{% set version = "11.1_20220318.memfault0" %}
+{% set version = "11.1_20220318" %}
+{% set release_url = "https://github.com/espressif/binutils-gdb/releases/download/esp-gdb-v{{ version }}" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/espressif/binutils-gdb/archive/refs/tags/esp-gdb-v11.1_20220318.tar.gz
-    sha256: 087e4ac9902da6ba894f22e7eb467a068256fff1b554095c6884adc9bdd701b7
-    folder: binutils-gdb-esp32-src
-    patches:
-      - backtrace-fix.patch
-      - fix-disassemble-segfault.patch
-  - git_url: https://github.com/espressif/crosstool-NG.git
-    git_rev: esp32-2019r1-rc2
-    folder: crosstool-NG-esp32
+  - url: {{ release_url }}/xtensa-esp-elf-gdb-{{ version }}-x86_64-apple-darwin14.tar.gz # [osx and x86_64]
+    sha256: 18324682734ca2a23a45c5505f7bc1824336f88a4d59327d545083dc8a633bba # [osx and x86_64]
+
+  # not available as of 2022-07-26
+  # - url: {{ release_url }}/xtensa-esp-elf-gdb-{{ version }}-x86_64-apple-darwin14.tar.gz # [osx and arm64]
+  #   sha256: fb613dacb25149f140f73fe9ff6c380bb43328e6bf813473986e9127e2bc283b # [osx and arm64]
+
+  - url: {{ release_url }}/xtensa-esp-elf-gdb-{{ version }}-x86_64-linux-gnu.tar.gz # [linux64]
+    sha256: c2bfb97c61fa3d2e3720aa1cc8163806a8bc712a173a374b60f6f5c061d33d3f # [linux64]
+
+  - url: {{ release_url }}/xtensa-esp-elf-gdb-{{ version }}-x86_64-w64-mingw32.zip # [win]
+    sha256: b1bff9ee6b2a0025f33a9ddc97b92bab7578c11172c06e38a7483f98e157fb93 # [win]
+
 
 build:
-  number: '2'
+  number: '3'
   skip: True  # [win]
 
 requirements:
-  build:
-    - {{ compiler('fortran') }}
-    - {{ compiler('c') }}  # [linux]
-    - {{ compiler('cxx') }}  # [linux]
-    - make
-    - bison # [linux]
   host:
```

Comment:
Can't you remove...almost all of these? There are no runtime dependencies of this package anymore - it's all baked into the binary or depends on the system Python to be installed.

Reply:
Unfortunately, I dont' really know which of these options are used and which ones aren't needed, but I was able to get less errors and crap going on if I defined the following:

```
build:
  number: '3'
  binary_relocation: False
  detect_binary_files_with_prefix: False
  ignore_prefix_files: True
  missing_dso_whitelist:                                     # [osx]
    - /Library/Frameworks/Python.framework/Versions/*/Python # [osx]
```

Reply:
:raised_eyebrow: on linux at least, there are several runtime deps required for the binary:
```bash
❯ which xtensa-esp-elf-gdb-3.8
/home/noah/mambaforge/envs/memfault/bin/xtensa-esp-elf-gdb-3.8

~/dev/memfault/memfault noahp/mflt-6636 !3                                                                                                                                        🌊 [memfault] 19:53:48 EDT
❯ ldd $(which xtensa-esp-elf-gdb-3.8)
        linux-vdso.so.1 (0x00007ffe7a9f1000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f1afc6c7000)
        libpython3.8.so.1.0 => /lib/x86_64-linux-gnu/libpython3.8.so.1.0 (0x00007f1afc179000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f1afc092000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f1afc08d000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f1afbe65000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f1afc6ed000)
        libexpat.so.1 => /lib/x86_64-linux-gnu/libexpat.so.1 (0x00007f1afbe32000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f1afbe16000)
```

Reply:
oic, yeah can probably nuke the host deps :+1:

Reply:
NOPE i'm a dummy, we do need them. they tell conda that we're building python-version-specific variants of the package, otherwise it doesn't assume python is related. `host` would normally be for a cross build purposes, informing `target` libs, but in our case we're still targeting native arch, just need to enumerate the python dependency at build time.